packages/emotion-theming/types/tests.tsx

```diff
@@ -54,7 +55,7 @@ const ThemedCompWithDefault = withTheme(CompCWithDefault)
 const {
   ThemeProvider: TypedThemeProvider,
   withTheme: typedWithTheme
-} = emotionTheming as emotionTheming.EmotionTheming<Theme>
+}: emotionTheming.EmotionTheming<Theme> = emotionTheming
```

Comment:
Found a test which used `EmotionTheming`, have added it's usage to the docs

Reply:
The added usage example uses `as` assertion, this doesn't. It would be better to keep consistency across those two places. I'm wondering though - why this here (without `as`) is allowed? Like TS can't guarantee that some kind of unknown Theme parameter (the one in `emotionTheming` here) is assignable to a "concrete" Theme defined here 🤔

Reply:
Yeah odd. Must be because of the default of `Theme` to `any` that it's assignable. At any rate, have synced this.